client/components/navbar.jsx

```diff
@@ -86,6 +86,11 @@ export default class Navbar extends React.Component {
     });
   }
 
+  handleWriteReview(event) {
+    event.preventDefault();
+    window.location.hash = '#review-form';
+  }
```

Comment:
I believe you can remove this now. I recommend that you change the other `<li>` elements the same way.